.github/send_alerts_to_slack.py

```diff
@@ -0,0 +1,95 @@
+"""
+
+Summarize the Frictionless Data JSON error output and if there are any errors, 
+send a summary message to a given Slack channel. Send a more detailed 
+error summary to std out and create a detailed txt file that can be accessed via GitHub
+artifacts.
+
+To run elsewhere, define SLACK_WEBHOOK in your repo's secrets and change the 
+top-level constants defined below. 
+
+`validate-csv.yml` can also be copied verbatim, assuming that the schema is 
+named `datapackage.json`.
+
+Links to...
+- Generate a Slack token
+    https://slack.com/apps/A0F7XDUAZ-incoming-webhooks
+- API for POST requests / formatting text:
+    https://api.slack.com/messaging
+
+
+"""
+
+import json
+import requests
+import os
+
+ERROR_FILE = 'error_summary.json'
+REPO = 'Princeton-CDH/pemm-data'
+CHANNEL = '#slack-webhook-test'
+ERROR_MAX = 5
```

Comment:
Since you're changing the others — would probably be nice if this one was configurable outside the script too; maybe an environment variable in the github action workflow ? or a command line option with a default?  (This one isn't as important as the others, though.)

Reply:
I've placed all configuration in the yml file. I'm pretty sure it's clearer that way, or I've just been staring at it for too long haha